### DISCUSSION TITLE
How long is this going to take?

### DISCUSSION THREAD

**Discussion by anthonyjclarke** [mtalcott/google-photos-deduper#53] [General]
Finally all installed and operational.... thanks to Mark and all those here who helped.

Been running for 9:30 and only 93/1539 completed.... last time it crashed out.....

<img width="920" alt="image" src="https://github.com/mtalcott/google-photos-deduper/assets/9491428/3faee236-3154-461e-8ca3-3ec9750d0c3a">

So, I'm wondering if all this looks right / ok / normal?

<img width="898" alt="image" src="https://github.com/mtalcott/google-photos-deduper/assets/9491428/8b9114f7-c780-4f7e-9d76-f608afc1e1db">

Thanks

**Comment by aleon1220** [comment]
i executed the instruction in my Lenovo L570 from 2018 using Linux ubuntu 22. 

1. try one set up + execution 50 mins
2. second try 40 mins for 89K+ photos.

My suggestion is to do it in Linux or if you want an adventure go for WSL running in Windows 10 or windows11
